core/mo-services-xml/src/main/resources/xml/ServiceDefPLATFORM.xml

```diff
@@ -1732,9 +1746,21 @@
           <mal:item comment="Reaction wheel w"
                     nvalue="6" value="WHEEL_W" />
         </mal:enumeration>
+        <mal:enumeration
+          comment="Attitude determination mode identifier."
+          name="AttitudeDeterminationMode" shortFormPart="13">
+          <mal:item comment="Attitude determination using Magnetic and Sun during pre-alignment"
```

Comment:
Suggest lower case naming. Furthermore it sounds enigmatic. Perhaps say `magnetometer and sun sensors`?

Reply:
Why does it say "during pre-alignment" and others do not? What does it mean? Unclear from the interface description.

Reply:
fixed